### PR TITLE
Feature/20 seed quick tip data

### PR DESCRIPTION
**Pull request by bryanakitchen**
Adds initial Quick Tip data to tables with initial data, create tables, drop tables, and load seed.

### CODE REVIEW

File: data/tips-data.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    tip: 'You\'ve been working hard! Take a lap around your office or home.'
+  },
+  {
+    tip: 'Take out a notepad and pen and write for a few minutes. Don’t worry if nothing brilliant comes to mind — even just writing down your feelings or what you\'re grateful for can be incredibly relaxing.'
+  },
+  {
+    tip: 'Close your eyes and take ten deep breaths.'
+  },
+  {
+    tip: 'Music break! Listen to one track that relaxes or energizes you. Dance it out if the feeling strikes you! [Consider headphones if you have coworkers nearby.]'
```
Comment (ShaneUP1) [inline data/tips-data.js:27]: Awesome!

**Review by ShaneUP1 — APPROVED** [review]
Awesome, thanks!

File: data/drop-tables.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+run();
+
+async function run() {
+
+  try {
+    await pool.connect();
+
+    await pool.query(`
+        DROP TABLE IF EXISTS links CASCADE;
```
Comment (franco-ortega) [inline data/drop-tables.js:11]: These two tables aren't joined, so I don't think CASCADE is necessary. But I'm not quite sure either way.

File: data/tips-data.js (hunk 1 above)
Comment (franco-ortega) [reply data/tips-data.js:27]: These are rad! I really like the level of detail provided.

**Review by franco-ortega — APPROVED** [review]

File: data/drop-tables.js (hunk 2 above)
Comment (bryanakitchen) [reply data/drop-tables.js:11]: Cool. Good point. Thanks!

**Pull request merged**